sql/src/test/java/io/crate/integrationtests/SubSelectIntegrationTest.java

```diff
@@ -713,4 +714,19 @@ public void testWhereSubsSelectAsClusteredByValue() {
         execute("select count(*) from t1 where r = (select r from t1 where id = 1)");
         assertThat(response.rowCount(), is(1L));
     }
+
+    public void testSubscriptOnSubSelect() {
+        execute("create table t1 (a object, c object)");
+        execute("insert into t1 (a, c) values ({ b = 1 }, { d = { e = 2 }})");
+        execute("refresh table t1");
+        execute("select a['b'], c['d']['e'] from (select * from t1) t2");
+        assertThat(printedTable(response.rows()), is("1| 2\n"));
+    }
+
+    @Ignore("Object literal inner value types are not parsed and so the fallback relying on object inner types won't work")
```

Comment:
Is there a reason why they aren't parsed - if so, can we add why as a comment or so?

Reply:
We discussed and agreed that we don't want to implement parsing of user-given object literals currently. If you changed your mind, I'm open for re-discussion.

Reply:
Ah right. Maybe clarify this with something like 

```suggestion
    @Ignore("We haven't added inner types for object literals, so this test is expected to fail")
```